### ISSUE TITLE
`。` (U+3002) as a proper punctuation

### ISSUE THREAD

**Issue by eggplants** [sindresorhus/awesome-lint#147]
In Japanese and Chinese, `。` (U+3002) is used as a common punctuation. Please see also: https://en.wikipedia.org/wiki/Japanese_punctuation#Full_stop

I would like to add `。` to `validateListItemSuffix` in `rules/list-item.js` as proper punctuation. Is it ok?

### Situation

I used awesome-lint to lint <https://github.com/Make-IT-TSUKUBA/awesome-tsukuba> (in Japanese) and then I got:

```shellsession
$ npx awesome-lint https://github.com/Make-IT-TSUKUBA/awesome-tsukuba
Need to install the following packages:
  awesome-lint
Ok to proceed? (y) y
✖ Linting

  README.md:1:1
  ✖   1:1    Missing file contributing.md                                 remark-lint:awesome-contributing
  ✖   1:1    Git repository must be at least 30 days old                  remark-lint:awesome-git-repo-age
  ✖   1:1    The repository should have "awesome" as a GitHub topic       remark-lint:awesome-github
  ✖   1:1    The repository should have "awesome-list" as a GitHub topic  remark-lint:awesome-github
  ✖   1:1    License was not detected by GitHub                           remark-lint:awesome-github
  ✖   1:1    Missing or invalid Table of Contents                         remark-lint:awesome-toc
  ✖  21:50   List item description must end with proper punctuation       remark-lint:awesome-list-item
(...same error was printed many times...)
  ✖  77:79   List item description must end with proper punctuation       remark-lint:awesome-list-item

  42 errors
```
